src/core/components.py

```diff
@@ -1042,25 +1043,31 @@ class Table(Component):
             a jsonpath to look up table data.
         table_headers_path: Str:
             a jsonpath to look up table headers.
+        table_container_width : Integer:
+            Number of pixels the table wrapper will be, if specified and table is wider than the container
+            Horizontal scrolling will be automatically shown on front end
     """
 
     __slots__ = [
         "table_components",
         "table_data_path",
         "table_headers_path",
+        "table_container_width",
```

Comment:
Q: I'm guessing it wasn't possible to make this work without specifying a 'forced width', as the table just shrinks the columns to try and fit as we add more?

Reply:
Coorect